pkg/apis/metal3/v1alpha1/baremetalhost_types.go

```diff
@@ -126,6 +127,39 @@ type BMCDetails struct {
 	CredentialsName string `json:"credentialsName"`
 }
 
+// RAIDVolume defines the desired configuration of volume in RAID
+type RAIDVolume struct {
+	// Size (Integer) of the logical disk to be created in GiB.
+	// If unspecified or set be 0, the maximum capacity of disk will be used for logical disk.
+	SizeGb int `json:"sizeGb,omitempty"`
```

Comment:
If the size is in GiB then let's call it `sizeGibibytes` not `sizeGb`.
We use `ramMebibytes` for the RAM size in HardwareDetails, so that would be consistent.